lib/src/validator/strict_dependencies.dart

```diff
@@ -2,49 +2,76 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import '../entrypoint.dart';
 import 'dart:async';
-import 'dart:io';
 
 import 'package:analyzer/analyzer.dart';
-import 'package:pub/src/solver/version_solver.dart';
+import 'package:path/path.dart' as path;
```

Comment:
`as p`. Pub is kind of inconsistent about this, but that's only because it's old—new files always use `as p`.